src/index.js

```diff
@@ -6,6 +6,9 @@
  * @license MIT
  */
 
+// Optimize for webpack UglifyJsPlugin
+global.__VERSION__ = global.__VERSION__ || false
```

Comment:
If we doesn't add this line, `dist` file will contain all data of the `package.json`.